src/main/kotlin/org/jitsi/jibri/selenium/pageobjects/CallPage.kt

```diff
@@ -154,6 +154,29 @@ class CallPage(driver: RemoteWebDriver) : AbstractPageObject(driver) {
         }
     }
 
+    /**
+     * Returns a count of how many remote participants are totally muted (audio
+     * and video).
+     */
+    fun numRemoteParticipantsMuted(): Int {
+        val result = driver.executeScript("""
+            try {
+                return APP.conference._room.getParticipants()
+                    .filter(participant => participant.isAudioMuted() && participant.isVideoMuted())
+                    .length;
+            } catch (e) {
+                return e.message;
+            }
+        """.trimMargin())
+        return when (result) {
+            is Number -> result.toInt()
+            else -> {
+                logger.error("error running numRemoteParticipantsMuted script: $result ${result::class.java}")
+                1
```

Comment:
Does the return of `numRemoteParticipantsMuted` equal `1` on error?

Reply:
this value was leftover from another implementation of this method (where even 1 didn't feel great but couldn't think of a better default value).  now that we know this is numRemoteParticipants, i think 0 would be better.  will change that.